user_install.sh

```diff
@@ -50,6 +50,7 @@ check_host() {
 #=======================#
 
 CATKIN_DIR=~/mil_ws
+MIL_CONFIG_DIR=~/.mil
 BASHRC_FILE=~/.bashrc
 MILRC_FILE=~/.milrc
```

Comment:
With Bash, it's convention to keep the rc files in `~/`. A few programs on my system have a similar setup, an rc file with a matching directory. I could move it inside if it bugs you, but I don't see it as a configuration file.